src/users/inputLotto.js

```diff
@@ -0,0 +1,35 @@
+import { Console } from "@woowacourse/mission-utils";
+import Validator from "../utils/ValidatorInput.js";
+
+export default class InputHandler {
+  async getPurchaseValue() {
+    const purchaseValue = await Console.readLineAsync(
+      "구입 금액을 입력해 주세요.\n"
+    ); // 문자열이므로 10진수로 변환
+    const parsedValue = parseInt(purchaseValue, 10);
+
+    if (Validator.validateAmount(parsedValue)) return parsedValue;
```

Comment:
모든 함수는 return값이 **명시적으로** 존재해야 해요.
다시 말하면, 조건문으로 return해 버리면 조건을 만족하지 못하는 경우 return값을 우리가 알 수 없어 큰 문제가 일어날 수 있어요.
아래 함수들도 아래와 같은 형식으로 고치면 좋을 것 같아요!
```suggestion
    Validator.validateAmount(parsedValue);
    return parsedValue;
```